graylog2-web-interface/src/views/components/widgets/MoveWidgetToTabModal.jsx

```diff
@@ -0,0 +1,64 @@
+// @flow strict
+import React, { useState } from 'react';
+import { ListGroup, ListGroupItem } from 'components/graylog';
+import BootstrapModalForm from 'components/bootstrap/BootstrapModalForm';
+import Input from 'components/bootstrap/Input';
+import { useStore } from 'stores/connect';
+import QueryTitle from 'views/logic/queries/QueryTitle';
+import { CurrentQueryStore } from 'views/stores/CurrentQueryStore';
+
+import View from 'views/logic/views/View';
+
+type Props = {
+  view: View,
+  widgetId: string,
+  onCancel: () => void,
+  onSubmit: (widgetId: string, selectedTab: ?string, keepCopy: boolean) => void,
+};
+
+type TabEntry = { id: string, name: string };
+
+const _tabList = (view: View): Array<TabEntry> => {
+  const queryIds = Object.keys(view.state.toObject());
+  return queryIds.map((queryId) => {
+    const tabTitle = QueryTitle(view, queryId) || 'Unknown Page title';
+    return ({ id: queryId, name: tabTitle });
+  });
+};
+
+const MoveWidgetToTabModal = ({ view, onCancel, onSubmit, widgetId }: Props) => {
+  const [selectedTab, setSelectedTab] = useState(null);
+  const [keepCopy, setKeepCopy] = useState(false);
+  const { id: activeQuery } = useStore(CurrentQueryStore);
+
+  const list = _tabList(view).filter(({ id }) => id !== activeQuery);
```

Comment:
Regarding the sort, it does not look correct to me, compared with the dashboard:
![image](https://user-images.githubusercontent.com/46300478/84006918-da784300-a96f-11ea-8ee9-18244788e33e.png)

One solution is using the order / values provided by the `QueryIdsStore`.

Reply:
With the last review I suggested to remove the current tab from the list, but after having another look I prefer to just disable the active tab. It has the benefits, that the user always sees the "same" list no matter on which tab he opens the modal and it is easier to find the surrounding tabs.

Reply:
Regarding the sort: https://github.com/Graylog2/graylog2-server/issues/8284 there is no order :( Which we need to fix before this PR I think now. :(

Regarding disable: I think you were right before, the user does not expect to find the current tab here. Therefore I would leave it out to reduce the visual noise. Let's discuss it later in or after the meeting.